### ISSUE TITLE
Missing skin tone aliases for Slack

### ISSUE THREAD

**Issue by viktorzetterstrom** [felldo/JEmoji#53]
### Your Question

Hello!

Thanks for all the work with this library! I have a question, we want to get slack aliases from JEmoji for storing reactions to messages. Unfortunately it seems emojis with skin tone do not have slack aliases. Why is that?

Best regards
Viktor

**Comment by felldo** [comment]
Hi, thank you making me aware of this issue. There are definitely emojis that do have a slack alias.
```json
{
  "emoji" : "👋🏻",
  "unicode" : "\uD83D\uDC4B\uD83C\uDFFB",
  "discordAliases" : [ ":wave_tone1:", ":wave::skin-tone-1:", ":waving_hand::skin-tone-1:" ],
  "githubAliases" : [ ],
  "slackAliases" : [ ":wave::skin-tone-2:" ],
  "hasFitzpatrick" : true,
  "hasHairStyle" : false,
  "version" : 1.0,
  "qualification" : "fully-qualified",
  "description" : "waving hand: light skin tone",
  "group" : "People & Body",
  "subgroup" : "hand-fingers-open",
  "hasVariationSelectors" : false
}
```
Appearently after I switched from loading emojis from a JSON file to having static generated Java code I mixed up the github and slack alias order. Therefore, currently github aliases are slack aliases and vice versa. In the meantime you can use the github aliases for your code while a new version is being released. This might take a couple of hours until it is available on mavencentral with the version `1.5.1`.

**Comment by viktorzetterstrom** [comment]
Great to hear! Thanks for the quick response and action! 🙏
